### ISSUE TITLE
 File "/Users/*****//projects/Real_Time_Image_Animation/env/lib/python3.7/site-packages/torch/cuda/__init__.py", line 75, in _check_driver     raise AssertionError("Torch not compiled with CUDA enabled") AssertionError: Torch not compiled with CUDA enabled

### ISSUE THREAD

**Issue by borodaalex** [anandpawara/Real_Time_Image_Animation#10]
**I've faced this in OSx - System Version:	macOS 10.14.6 (18G103)**

 File "/Users/*****/projects/Real_Time_Image_Animation/env/lib/python3.7/site-packages/torch/cuda/__init__.py", line 161, in _lazy_init
    _check_driver()
  File "/Users/*****/projects/Real_Time_Image_Animation/env/lib/python3.7/site-packages/torch/cuda/__init__.py", line 75, in _check_driver
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled

**Comment by AshwinRaikar88** [comment]
Yes you can use it, on cpu  (But the output will be very slow)

Now here's how to do it, ( you'll need to change two files)

In line 40 of image_animation.py, set
`cpu = True`

In line 24 of demo.py, set
`def load_checkpoints(config_path, checkpoint_path, cpu = True)`

**Comment by Serafim-End** [comment]
and firstly use right torch version 
for mac it is 
`pip install torch torchvision`

**Comment by borodaalex** [comment]
I believe this should be done by 

pip install torch===1.0.0 torchvision===0.2.1 -f https://download.pytorch.org/whl/cu100/torch_stable.html

from instructions of project

**Comment by anandpawara** [comment]
> and firstly use right torch version
> for mac it is
> `pip install torch torchvision`

please do install correct torch version as latest version for torch and torchvision does is  by this project

**Comment by pardhu-mallela** [comment]
> 

Yes. With cpu, output is very slow. I am unable to run this code in google colab because colab is not supporting webcam directly. Is there any alternative to get the output quicky?

**Comment by pardhu-mallela** [comment]
> Yes you can use it, on cpu (But the output will be very slow)
> 
> Now here's how to do it, ( you'll need to change two files)
> 
> In line 40 of image_animation.py, set `cpu = True`
> 
> In line 24 of demo.py, set `def load_checkpoints(config_path, checkpoint_path, cpu = True)`
